buildVerifier.py

```diff
@@ -2,6 +2,9 @@
 import platform
 import subprocess
 import re
+import glob
```

Comment:
Glob is unix command what about Windows?

Reply:
Glob has unix style syntax command but is generally system agnostic